src/components/tasks/card/index.tsx

```diff
@@ -258,13 +262,171 @@ const Card: FC<Props> = ({
+                        <div className={classNames.spinner}>
+                            <span className={classNames.screenReaderOnly}>
+                                loading
+                            </span>
+                        </div>
+                    </div>
+                )}
+
+                <div className={classNames.cardItems}>
+                    <h2
+                        className={classNames.cardTitle}
+                        contentEditable={shouldEdit}
+                        onKeyPress={(e) => handleChange(e, 'title')}
+                        role="button"
+                        tabIndex={0}
+                    >
+                        {cardDetails.title}
+                    </h2>
+                    {/* progress bar */}
+                    <div className={classNames.progressContainerUpdated}>
+                        <div className={classNames.progressIndicator}>
+                            <div
+                                className={`
+                ${handleProgressColor(
+                    content.percentCompleted,
+                    content.startedOn,
+                    content.endsOn
+                )}
+                ${classNames.progressStyle}
+              `}
+                                style={{
+                                    width: `${content.percentCompleted}%`,
+                                }}
+                            ></div>
+                        </div>
+                        <span>{content.percentCompleted}% </span>
+                    </div>
+                </div>
+                <div className={classNames.dateInfo}>
+                    <div>
+                        <span className={classNames.cardSpecialFont}>
+                            Estimated completion{' '}
+                        </span>
+                        {renderDate(fromNowEndsOn, shouldEdit)}
+                    </div>
+                    <span
+                        className={classNames.cardSpecialFont}
+                        contentEditable={shouldEdit}
+                        onKeyPress={(e) => handleChange(e, 'startedOn')}
+                        role="button"
+                        tabIndex={0}
+                    >
+                        {cardDetails.status == 'Available'
+                            ? 'Not started '
+                            : `Started on ${fromNowStartedOn}`}
+                    </span>
+                </div>
+
+                <div className={classNames.cardItems}>
+                    <div className={classNames.contributor}>
+                        <span className={classNames.cardSpecialFont}>
+                            Assigned to{' '}
+                        </span>
+                        <span className={classNames.contributorImage}>
+                            <Image
+                                src={assigneeProfilePic}
+                                alt="Assignee profile picture"
```

Comment:
This would be a better alt text
```suggestion
                                alt="{`profile picture of ${{cardDetails.assignee}}`}"
```